bundles/com.salesforce.bazel.eclipse.core/src/com/salesforce/bazel/eclipse/core/model/discovery/projects/JavaSourceInfo.java

```diff
@@ -763,7 +764,7 @@ private void reportDeltaAsProblem(MultiStatus result, Path rootDirectory,
                     .collect(joining("\n - ", " - ", "\n"));
         }
         result.add(
-            Status.error(
+            Status.warning(
```

Comment:
Warnings are useless. They tend to be ignored by users in large projects. I prefer to keep having this as an error because in 99.9% of the cases a Bazel workspace with these problems will not be fully usable in Eclipse/JDT. 

It's a split package scenario but in the same physical folder. Thus, most likely this is an indication that developers should investigate the `project-per-package` strategy and moving things to different Java packages for better Eclipse/JDT support. It's unfortunate but the flexibility of Bazel is not well support by JDT.

Reply:
If we keep this as an error, we won't be able to import basic java tutorial described at [Bazel Tutorial: Build a Java Project](https://bazel.build/start/java) no matter if we use the project-per-package or project-per-target strategy.

Reply:
The error happens when importing and run the Sync Bazel Projects View action.